protocol-designer/src/labware-ingred/actions/thunks.ts

```diff
@@ -1,29 +1,25 @@
-// @flow
 import assert from 'assert'
 import { getIsTiprack } from '@opentrons/shared-data'
 import { uuid } from '../../utils'
 import { selectors as labwareDefSelectors } from '../../labware-defs'
 import { selectors as stepFormSelectors } from '../../step-forms'
 import { selectors as uiLabwareSelectors } from '../../ui/labware'
 import { getNextAvailableDeckSlot, getNextNickname } from '../utils'
-
-import type {
+import {
   CreateContainerArgs,
   CreateContainerAction,
   DuplicateLabwareAction,
 } from './actions'
-import type { ThunkAction } from '../../types'
-
-export type RenameLabwareAction = {
-  type: 'RENAME_LABWARE',
+import { ThunkAction } from '../../types'
+export interface RenameLabwareAction {
+  type: 'RENAME_LABWARE'
   payload: {
-    labwareId: string,
-    name?: ?string,
-  },
+    labwareId: string
+    name?: string | null | undefined
```

Comment:
should be the same as this
```suggestion
    name?: string | null
```